objcgen/driver.cs

```diff
@@ -549,6 +549,9 @@ public int Compile ()
 						mtouch.Append ("--dsym:false ");
 						mtouch.Append ("--msym:false ");
 						mtouch.Append ($"--embeddinator ");
+						// FIXME: we need to generate this XML file, issue #301
+						// https://github.com/mono/Embeddinator-4000/issues/301
+						mtouch.Append ($"--xml={Path.Combine (OutputDirectory, "linker.xml")} ");
```

Comment:
^ needs a call to `Quote` or it will fails if `OutputDirectory` has any space in it